src/vector/ops/set.rs

```diff
@@ -32,6 +32,35 @@ impl Geometry {
         }
         Some(unsafe { Geometry::with_c_geometry(ogr_geom, true) })
     }
+
+    /// Compute union.
+    ///
+    /// Generates a new geometry which is the region of union of
+    /// the two geometries operated on.
+    /// Geometry validity is not checked. In case you are unsure of the
+    /// validity of the input geometries, call IsValid() before,
+    /// otherwise the result might be wrong.
+    ///
+    /// # Returns
+    /// Some(Geometry) if both Geometries contain pointers
+    /// None if either geometry is missing the gdal pointer, or there is an error.
```

Comment:
```suggestion
    /// `None` if either geometry is missing the gdal pointer, or there is an error.
```

Reply:
I note that the documentation for `intersection` isn't that great. The docs in `predicates.rs` is closer to the mark. Would be great if you can bring it to that level, but if you can't I'll do a separate PR to normalize it.